internal/pkg/config/config.go

```diff
@@ -318,14 +319,13 @@ func ReadConfig() (*Config, error) {
 }
 
 func readConfigBytes() (*Config, error) {
-	if configFromBytes == nil {
-		unmarshalled := new(Config)
-		unmarshalledErr := yaml.UnmarshalStrict(configBytes, unmarshalled)
-		if unmarshalledErr != nil {
-			fmt.Println(unmarshalledErr)
+	configBytesOnce.Do(func() {
+		configFromBytes = new(Config)
```

Comment:
nit: config and configError instead of configFromBytes and configFromBytesError.

Reply:
I agree that these names are a bit verbose, but I want to avoid any potential confusion with the config that is read from a file. WDYT?

Reply:
SGTM.